setup.py

```diff
@@ -22,11 +22,14 @@
     ],
     license='New BSD',
     classifiers=[
-        'Development Status :: 4 - Beta',
+        'Development Status :: 2 - Pre-Alpha',
```

Comment:
Note: The badges won't update on the main README until the setup.py changes are pushed up to pypi.